packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,7 +302,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: dropdownWidth || this._dropDown.clientWidth - 2 } }>
```

Comment:
well rabbit hole...I'm thinking of pushing the custom width to the callout to make it easier to pass to the proper place. This also would mean that callouts could have defined widths. or else we just glamor'ize it all and fergetaboutit